src/performance/index.ts

```diff
@@ -0,0 +1,148 @@
+import path from 'path';
```

Comment:
Could you add a REAMD to teach us how to run the performance?

My guess now is run this file with `node`. Is it true?

Even better, you can put the performance result with the running environment info.

Reply:
Of course, let me add a new Commit